src/eko/evolution_operator/__init__.py

```diff
@@ -151,9 +152,11 @@ def quad_ker(
         Mellin inversion point
     areas : tuple
         basis function configuration
-    a1 : float
+    as1 : float
         target coupling value
-    a0 : float
+    as_raw : float
+        coupling value at the process scale
```

Comment:
Move from here, such that `as_raw` is not between `as0` and `as1`:
- the first one corresponds to the point at which it has to be evaluated for the expanded part of the predictions (the "as value")
- the latter two are the boundaries of the running (so the "as function"), the way it is used for DGLAP solution (and it is there only because we are trading the fact scale dependency for an as dependency through the "as function", but it is not related in any way to the as used for the process)